plugins/source/aws/client/multiplexers.go

```diff
@@ -32,7 +32,12 @@ func AccountMultiplex(meta schema.ClientMeta) []schema.ClientMeta {
 			region := getRegion(client.ServicesManager.services[partition][accountID])
 			// Ensure that the region is always set by a region that has been initialized
 			if region == "" {
-				meta.(*Client).Logger().Trace().Str("accountID", accountID).Str("partition", partition).Msg("no valid regions have been specified for this account")
+				if !client.allRegions {
```

Comment:
Can we do `if client.allRegions` instead of inverted logic (`if !client.allRegions`)?

Reply:
Changed the name so its not inverted